### DISCUSSION TITLE
wrfoutput error

### DISCUSSION THREAD

**Discussion by neha12-3** [NCAR/hrldas#179] [Q&A] [answered]
i have got wrfout file but during run this error occured
![Uploading 1714971330332.jpg…]()

**Comment by cenlinhe** [comment, marked as answer]
your picture is not uploaded successfully, so we cannot see anything. Also, for all future WRF-related questions, please use the WRF discussion forum: https://forum.mmm.ucar.edu/
You may also find that a lot of your questions have already been answered in the WRF forum. It is a great platform.

**Comment by neha12-3** [comment]
have got wrfout file but during run this error occured
![Uploading 1730106026678.jpg…]()

**Comment by neha12-3** [comment]
![1730106026678](https://github.com/user-attachments/assets/1f7b6bcc-1c96-4eab-8efc-0a041e4e23e5)
got this error after running OMP_NUM_THREADS command nd tail command for wrfout

**Reply by cenlinhe** [reply]
did you include enough data in the wrfbdy_d01? it looks like you only include 1-day data in wrfbdy_d01 for 2018-01-01.

**Comment by neha12-3** [comment]
![1730965050623](https://github.com/user-attachments/assets/cfe14b45-0ad6-4de2-8572-cd51f75e7184)
facing error after running wrf.exe

**Comment by neha12-3** [comment]
i am trying to create 3 months ctl and dat file from arwpost but after 1 month it is showing an error running out of input data and then stop

**Reply by tslin2** [reply]
You may post this to https://forum.mmm.ucar.edu/ or https://github.com/wrf-model/WRF which is more related to WRF.
Thanks!
